lms/djangoapps/grades/tasks.py

```diff
@@ -32,61 +32,37 @@ def recalculate_subsection_grade(user_id, course_id, usage_id, only_if_higher):
         be updated only if the new grade is higher than the previous
         value.
     """
-    if not PersistentGradesEnabledFlag.feature_enabled(course_id):
+    course_key = CourseLocator.from_string(course_id)
+    if not PersistentGradesEnabledFlag.feature_enabled(course_key):
```

Comment:
This line changing confuses me, as I thought the `feature_enabled` check always takes a course id. Had this been incorrect before?